test/judges/test-add-review-comments.rb

```diff
@@ -71,6 +71,26 @@ def test_adds_review_comments_to_facts_without_review_comments
     end
   end
 
+  def test_handles_not_found_repo
+    WebMock.disable_net_connect!
+    pl = { id: 93, comments: 2 }
+    repo = 90
+    stub(repo, pl)
+    what = 'pull-was-reviewed'
+    fb = Factbase.new
+    fact = fb.insert
+    fact.what = what
+    fact.issue = pl[:id]
+    fact.repository = repo
+    load_it('add-review-comments', fb)
+    facts = fb.query("(eq what \"#{what}\")").each.to_a
+    assert_equal(pl[:id], facts.first.issue)
+    assert_equal(
```

Comment:
@Suban05 in general, it's a bad idea to assert on error messages. Instead, check for `facts.first['review_comments'].nil?`